pakyow-test/lib/test_help/ext/request.rb

```diff
@@ -1,7 +1,9 @@
 module Pakyow
   class Request
-    def params
-      env['pakyow.params']
+    old_params = instance_method(:params)
+
+    define_method :params do
```

Comment:
Oh, gotcha the local variable wouldn't be in scope of `def`. Yeah maybe `alias_method` would do it. I feel like that'd be a little clearer.

```ruby
alias_method :old_params, :params
def params
  env.fetch("pakyow.params") { old_params }
end
```

Does that do it?